### PR TITLE
initial import chmobil layers

### PR DESCRIPTION
**Pull request by ltclm**
add schweizmobil layers:
* ch.astra.wanderland
* ch.astra.veloland
* ch.astra.skatingland
* ch.astra.mountainbikeland

### CODE REVIEW

**Comment by ltclm** [comment]
Needs to be merged too:
https://github.com/geoadmin/service-sphinxsearch/pull/242

**Comment by ltclm** [comment]
[Testurl](https://mf-geoadmin3.dev.bgdi.ch/ltclm/?lang=de&topic=ech&bgLayer=ch.swisstopo.swissimage&layers=ch.astra.wanderland,ch.astra.veloland,ch.astra.skatingland,ch.astra.mountainbikeland,ch.swisstopo.lubis-luftbilder_schraegaufnahmen&layers_timestamp=,,,,&X=186392.50&Y=638765.00&zoom=6)

File: chsdi/models/vector/uvek.py [hunk 1]
```diff
@@ -2243,3 +2243,67 @@ class suel_pl_r_kraft (Base, Vector):
                                dimension=2, srid=21781))
 
 register('ch.bfe.sachplan-uebertragungsleitungen_kraft', suel_pl_r_kraft)
+
+
+class chmobil_veloland (Base, Vector):
```
Comment (procrastinatio) [inline chsdi/models/vector/uvek.py]: I think we use the camelCase convention for class name

**Comment by procrastinatio** [comment]
Try optimize PNG...

**Comment by ltclm** [comment]
pngs have been optimized and chmobil classnames changed to camelcase.

**Pull request merged**